### PR TITLE
Use (<p>) to split the text to different paragraphs in Javadoc comments.

### PR DESCRIPTION
**Pull request by arturobernalg**
See https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#format

### CODE REVIEW

**Comment by garydgregory** [comment]
@arturobernalg Close HTML tags please.

**Comment by arturobernalg** [comment]
> @arturobernalg Close HTML tags please.

why? 
its just a paragraph separator.

**Comment by garydgregory** [comment]
It is cleaner to follow the same element tag rules everywhere and close all tags.

**Comment by arturobernalg** [comment]
> It is cleaner to follow the same element tag rules everywhere and close all tags.

ok, so I do something like ```<p></p>```??

**Comment by garydgregory** [comment]
> > It is cleaner to follow the same element tag rules everywhere and close all tags.
> 
> ok, so I do something like `<p></p>`??

Yes, just like any other tag like em, code, pre, and so on.

**Comment by arturobernalg** [comment]
> > > It is cleaner to follow the same element tag rules everywhere and close all tags.
> > 
> > 
> > ok, so I do something like `<p></p>`??
> 
> Yes, just like any other tag like em, code, pre, and so on.

done

File: src/main/java/org/apache/commons/lang3/ObjectUtils.java [hunk 1]
```diff
@@ -984,7 +984,7 @@ public static boolean isArray(final Object object) {
 
     /**
      * Checks if an Object is empty or null.
-     *
+     * <p></p>
```
Comment (garydgregory) [inline src/main/java/org/apache/commons/lang3/ObjectUtils.java:987]: Uh?

Comment (garydgregory) [reply src/main/java/org/apache/commons/lang3/ObjectUtils.java:987]: `<p>`This is a paragraph.`</p>`

Comment (arturobernalg) [reply src/main/java/org/apache/commons/lang3/ObjectUtils.java]: done

Comment (garydgregory) [reply src/main/java/org/apache/commons/lang3/ObjectUtils.java]: Did you not see my example?

Comment (garydgregory) [reply src/main/java/org/apache/commons/lang3/ObjectUtils.java]: Are you not trying a local build before you push to git?

Comment (garydgregory) [reply src/main/java/org/apache/commons/lang3/ObjectUtils.java]: @arturobernalg 
`<p>`This is a paragraph.`</p>`, you can't embed lists per the errors in the build. Again, run a 'mvn' build before you push and you will discover this type of issue without forcing reviewers to ping you for broken builds. TY.

**Review by garydgregory — CHANGES_REQUESTED** [review]
@arturobernalg Please check GHA builds after you push, notice the failures.

**Pull request closed without merging**